projects/packages/jetpack-mu-wpcom/tests/php/features/launchpad/class-launchpad-task-lists-test.php

```diff
@@ -177,4 +177,64 @@ public function test_task_list_is_completed( $tasks, $expected, $task_list_optio
 		);
 		$this->assertEquals( $expected, wpcom_launchpad_is_task_list_completed( 'task-list-test' ) );
 	}
+
+	/**
+	 * Test the validation of the repetition props. Both properties(target_repetitions & repetition_count_callback)
+	 * must be set if one is set.
```

Comment:
Nit: I think we can simplify the description a bit.
```suggestion
	 * Test the validation of the repetition props.
	 *  Both target_repetitions and repetition_count_callback must be set if either property is set.
```